src/site/content/en/blog/disable-mouse-acceleration/index.md

```diff
@@ -0,0 +1,178 @@
+immersive web games. When the pointer is locked, all mouse events go to the
+target element of the pointer lock.
+
+Call `requestPointerLock()` on the target element to request a pointer lock, and
+listen to `pointerlockchange` and `pointerlockerror` events to monitor pointer
+lock changes.
+
+```js
+const myTargetElement = document.body;
+
+// Call this function to request a pointer lock.
+function requestPointerLock() {
+  myTargetElement.requestPointerLock();
+}
+
+document.addEventListener("pointerlockchange", () => {
+  if (document.pointerLockElement) {
+    console.log(`pointer is locked on ${document.pointerLockElement}`);
+  } else {
+    console.log("pointer is unlocked");
+  }
+});
+
+document.addEventListener("pointerlockerror", () => {
+  console.log("pointer lock error");
+});
+```
+
+### Disable mouse acceleration {: #disable-mouse-acceleration }
+
+Call `requestPointerLock()` with  `{ unadjustedMovement: true }` to disable
+OS-level adjustment for mouse acceleration, and access raw mouse input.
+This way, mouse movement data from `mousemove` events won't include mouse
+acceleration when the pointer is locked.
+
+Use the new returned promise from `requestPointerLock()` to know if the request
+was successful.
+
+```js
+function requestPointerLockWithUnadjustedMovement() {
+  const promise = myTargetElement.requestPointerLock({
+    unadjustedMovement: true,
+  });
+
+  if (!promise) {
+    console.log("disabling mouse acceleration is not supported");
+    return;
+  }
+
+  return promise
+    .then(() => console.log("pointer is locked"))
+    .catch((error) => {
+      if (error.name === "NotSupportedError") {
+        // Some platforms may not support unadjusted movement.
+        // You can request again a regular pointer lock.
+        return myTargetElement.requestPointerLock();
+      }
+    });
+}
+```
+
+It is possible to toggle between accelerated and non-accelerated mouse movement
+data without releasing the pointer lock. Simply request the pointer lock again
+with the desired option. If that request fails, the original lock will remain
+intact and the returned promise will reject. No pointer lock events will fire
+for a failed change request.
+
+### Browser support {: #browser-support }
+
+The Pointer Lock API is [well supported across browsers]. However Chromium-based
+browsers (e.g. Chrome, Edge, etc.) are the only ones to support disabling
+OS-level adjustment for mouse acceleration as of October 2020.
+See MDN's [Browser compatibility] table for updates.
+
+### Operating system support {: #os-support }
+
+Disabling OS-level adjustment for mouse acceleration is supported on Chrome OS,
+macOS Catalina 10.15.1, and Windows. Linux will follow.
+
+## Sample  {: #sample }
+
+You can play with the Pointer Lock API by running the [sample] on Glitch. Be
+sure to [check out the source code].
+
+{% Glitch { id: 'unadjusted-movement', path: 'script.js' } %}
```

Comment:
Suggested app changes:

Buttons: 
"request pointer lock"
"request unadjustedMovement lock"

Readouts:
Have a set of readouts for accelerated, and unadjusted, movement so that they can be compared.  Probably only 'x' data is needed to simplify.

Maybe have a 'clear' button?

Reply:
I'll update sample to address your feedback.

Reply:
Done